libkbfs/config_local.go

```diff
@@ -897,9 +898,16 @@ func (c *ConfigLocal) EnableJournaling(
 	log := c.MakeLogger("")
 	branchListener := c.KBFSOps().(branchChangeListener)
 	flushListener := c.KBFSOps().(mdFlushListener)
+	// Set the journal disk limit to 10 GiB for now. TODO: Make
+	// this configurable.
+	var journalDiskLimit int64 = 10 * 1024 * 1024 * 1024
```

Comment:
Yeah, my plan is to do something based on the disk size, but in a future PR. (probably the next one) That one will probably let me calculate a meaningful inode limit, too.

100 GB sounds like a lot to me, at least for now, but I'm not opposed to ending up with that. I'd prefer to start small and increase later, as that is easier than starting large.